qiskit_nature/second_q/algorithms/excited_states_solvers/excited_states_solver.py

```diff
@@ -58,9 +58,10 @@ def get_qubit_operators(
     ) -> Tuple[PauliSumOp, Optional[dict[str, PauliSumOp]]]:
         """Construct qubit operators by getting the second quantized operators from the problem
         (potentially running a driver in doing so [can be computationally expensive])
-        and using a QubitConverter to map + reduce the operators to qubit ops
+        and using a QubitConverter to map + reduce the operators to qubit operators.
```

Comment:
```suggestion
        and using a QubitConverter to map and reduce the operators to qubit operators.
```
Please also check if this same change can be applied in similar classes to this one